pbcommand/models/common.py

```diff
@@ -176,6 +176,7 @@ class MimeTypes(object):
     CSV = 'text/csv'
     XML = 'application/xml'
     BINARY = 'application/octet-stream'
+    PICKLE = 'application/python-pickle'
```

Comment:
I'm using whatever the default protocol is.  I'm hoping these files only
ever get used internally anyway - they are undocumented and only useful for
passing structured data between tasks at present.

On Sat, Sep 5, 2015 at 2:04 PM, Michael <notifications@github.com> wrote:

> In pbcommand/models/common.py
> <https://github.com/PacificBiosciences/pbcommand/pull/25#discussion_r38814788>
> :
>
> > @@ -176,6 +176,7 @@ class MimeTypes(object):
> >      CSV = 'text/csv'
> >      XML = 'application/xml'
> >      BINARY = 'application/octet-stream'
> > +    PICKLE = 'application/python-pickle'
>
> Which pickle protocol are you using? I believe the default is ascii.
>
> Systems that don't natively understand application/python-pickle will drop
> to application/octet- stream which may or may not be consistent with the
> pickle protocol version (i.e., using protocol 2)
>
> Can you clarify?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PacificBiosciences/pbcommand/pull/25/files#r38814788>.
>